### ISSUE TITLE
Enabling audio normalisation doesn't normalise the audio in some files

### ISSUE THREAD

**Issue by Vollpflock** [Mattk70/Chirpity-Electron#102]
**Is your feature request related to a problem? Please describe.**
Very quiet and noisy exported audio clips. 

**Describe the solution you'd like**
Apply audio preferences (normalization/volume adjustment, high pass filter, low shelf filter and attenuation) on export. At the moment, these are only audible in the Chirpity UI, and are not applied to exported clips.

**Describe alternatives you've considered**
Exporting as wav and editing in Audacity. Or API upload to Xeno-Canto ;)

**Additional context**
This has already been mentioned under https://github.com/Mattk70/Chirpity-Electron/discussions/60.

**Comment by Mattk70** [comment]
Actually, if the filters are enabled when you export, they are applied to the exported audio. With the notable exception of 'normalise'. 

When I think about this, I suspect normalise is the only one that _should_ be applied! Otherwise, when you open an exported file with gain / low shelf filters set as before, these filters are applied twice over. If you normalise a file twice, it stays normalised...

**Comment by Vollpflock** [comment]
And with the second notable exception of "Volume adjustment"? :)

Nah I really appreciate how the high pass filter, low shelf filter, and attenuation are applied during export. It significantly improves the sound quality when sharing files.

**Comment by Mattk70** [comment]
Ah, yes. I don't do volume adjustment either. That was probably because of the twice filtered issue I mentioned. It hurt my ears!

**Comment by Vollpflock** [comment]
Yeah, most of my files don't get fully normalized (at least that's what I call it; they do get somewhat normalized but still aren't louder), so I've set the volume adjustment to +50 dB for now.

**Comment by Vollpflock** [comment]
If feasible, could you also do this for 'Volume Adjustment' so that the Volume Adjustment filter is applied to exported audio, if enabled and active?

**Comment by Mattk70** [comment]
I'd prefer to get to the bottom of why normalisation doesn't do what it's supposed to for your files! If Audacity does it right, there must be another ffmpeg setting I'm not familiar with

**Comment by Vollpflock** [comment]
Yeah not sure what it is, but Audacity actually normalizes them.

**Comment by Vollpflock** [comment]
Thanks, here's the profile
![Screenshot 2024-10-26 at 12 26 49](https://github.com/user-attachments/assets/e0d6dfdb-1b03-4b75-975c-139dfcd0641f)

**Comment by Vollpflock** [comment]
Thank you. Yeah, it’s normalized now—not as loud as it needs to be, but at least it’s louder. But I can’t run this on every file; it just takes too long.

I don’t really understand how normalization works. If my file is an hour long and there’s a loud noise, does that mean the rest will be “quieter”?

And applying volume adjustment on export is a lot or difficult to code?

**Comment by Mattk70** [comment]
Finally! I have figured out why this is. If the maximum audio level in the file is too low, the loudness normalisation does not kick in. The solution is to increase the volume adjustment (with "Normalise Audio" enabled). Once the volume reaches a sufficient threshold, the file will be normalised
